### PR TITLE
model and controller of the villa

### PR DESCRIPTION
**Pull request by nnevedm**
I did the model, validates, controller and the index view of the villa

I'm going to do the show, new and create view right now

### CODE REVIEW

**Comment by nnevedm** [comment]
Capture d’écran 2024-02-27 à 14 44 55

view of the index villas

File: app/models/villa.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Villa < ApplicationRecord
+  belongs_to :user
+  validates :name, presence: true
```
Comment (CaroDeuse) [inline app/models/villa.rb]: do we also want uniqueness?

File: app/models/villa.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  validates :address, presence: true
+  validates :movie_genre, presence: true
```
Comment (CaroDeuse) [inline app/models/villa.rb]: do we already we to add only certain proposition available? eg horror, comedy, ...
maybe we don't need now, but we will need it later, if you don't do it now, then open a task in the backlog so we don't forget

File: db/migrate/20240227113858_create_villas.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class CreateVillas < ActiveRecord::Migration[7.1]
+  def change
+    create_table :villas do |t|
+      t.string :name
+      t.string :address
+      t.string :movie_genre
+      t.string :description
```
Comment (gdecooman) [inline db/migrate/20240227113858_create_villas.rb]: I wonder if text wouldn't be more appropriate for a description

**Review by gdecooman — COMMENTED** [review]
Finished the review; raised one question

File: app/models/villa.rb (hunk 2 above)
Comment (nnevedm) [reply app/models/villa.rb]: i will see how to do it

File: db/migrate/20240227113858_create_villas.rb (hunk 3 above)
Comment (nnevedm) [reply db/migrate/20240227113858_create_villas.rb]: done !

**Comment by nnevedm** [comment]
it's done ! have a look plz

File: app/models/villa.rb (hunk 2 above)
Comment (nnevedm) [reply app/models/villa.rb]: ✅

**Review by CaroDeuse — APPROVED** [review]

**Pull request merged**